src/components/date.js

```diff
@@ -28,7 +28,7 @@ export default class FormattedDate extends Component {
             return children(formattedDate);
         }
 
-        return <span>{formattedDate}</span>;
+        return <time datetime={value}>{formattedDate}</time>;
```

Comment:
Why not `<date>` here? Also the `datetime` attribute requires a specific format, and for the value to be a string.